justfile

```diff
@@ -150,6 +150,10 @@ genesis-sp1-ics07: build-sp1-programs
   @echo "Generating the genesis file..."
   RUST_LOG=info cargo run --bin operator --release -- genesis -o scripts/genesis.json
 
+# Deploy scripts. If these fail to land due to gas price fluctuations, try again with a higher gas price:
+# adding something like `--legacy --with-gas-price 100gwei` -- depending on the current gas prices.
+
+
```

Comment:
Kinda feel like all these scripts should be removed since after the security exercises tomorrow, I think we won't be using them from the justfile ever

Reply:
Not in prod, but they are sure useful in testnet :) But that might get moved to a CI pipeline or something eventually.